ppr-ui/tests/unit/TableRow.spec.ts

```diff
@@ -538,6 +538,7 @@ describe('Mhr TableRow tests', () => {
           break
         case MhApiStatusTypes.DRAFT:
           expect(rowData.at(3).text()).toContain(MhUIStatusTypes.DRAFT)
+          expect(rowData.at(rowData.length - 1).text()).toContain('Edit')
```

Comment:
I don't believe that at this point we are considering authorization when showing the Edit button, might need to update that.